### ISSUE TITLE
fails to build

### ISSUE THREAD

**Issue by droundy** [djc/template-benchmarks-rs#16]
Not sure what is going on here, but for some reason the download of `yarte` is failing.
```
template-benchmarks-rs$ cargo build
    Updating git repository `https://github.com/djc/askama`
    Updating crates.io index
    Updating git repository `https://github.com/krdln/fomat-macros`
    Updating git repository `https://github.com/sunng87/handlebars-rust`
    Updating git repository `https://github.com/Stebalien/horrorshow-rs`
    Updating git repository `https://github.com/cobalt-org/liquid-rust`
    Updating git repository `https://github.com/utkarshkukreti/markup.rs`
    Updating git repository `https://github.com/Keats/tera`
    Updating git repository `https://github.com/rust-iendo/yarte`
error: failed to load source for a dependency on `yarte`

Caused by:
  Unable to update https://github.com/rust-iendo/yarte

Caused by:
  failed to fetch into /home/droundy/.cargo/git/db/yarte-31dc0d46a4fad725

Caused by:
  failed to authenticate when downloading repository
attempted to find username/password via git's `credential.helper` support, but failed

Caused by:
  failed to acquire username/password from local configuration
```

**Comment by droundy** [comment]
Looks like the github repository has been deleted.

**Comment by djc** [comment]
Fixed in #17.
